velox/dwio/parquet/reader/CMakeLists.txt

```diff
@@ -12,8 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(${VELOX_ENABLE_QPL})
+  set(CompressionQpl "CompressionQpl.cpp")
+  set(CompressionQplDependency "qpl")
+else()
+  set(CompressionQpl "")
+  set(CompressionQplDependency "")
+endif()
+
```

Comment:
Please avoid using variables for target related things. You can use [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html) to conditionally add sources to an existing target and just call `target_link_libraries` again to add any dependencies.

Reply:
Thanks, I have completed it. Please help to review